tasks/failover_ip.yml

```diff
@@ -0,0 +1,21 @@
+- name: Add failover network interface alias
+  template:
+    src=network/interfaces.d/failover-ip
+    dest="{{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
+    owner=root
+    group=root
+    mode=0644
+
+- name: Ensure failover network is included in configuration file
+  lineinfile:
+    dest=/etc/network/interfaces
+    state=present
+    line="source {{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
```

Comment:
The file used to include

    source-directory /etc/network/interfaces.d

by default, but apparently that's not the case anymore (at least on my own machine and the LB server).